examples/chip-tool/commands/payload/Commands.h

```diff
@@ -0,0 +1,38 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "ParseCommand.h"
+
+class PayloadParse : public ParseCommand
+{
+public:
+    PayloadParse() : ParseCommand("parse") {}
+};
```

Comment:
I think ParseCommand is not something we will reuse much. Can you merge `PayloadParse` and `ParseCommand` ?

Reply:
Make sense, PayloadParse remove in the new revision.